internal/endorser/endorser_test.go

```diff
@@ -44,7 +45,10 @@ func TestGenerateEndorsement_SingleValidEndorsement(t *testing.T) {
 	}
 	tempURI := "file://" + tempPath
 	provenances := []string{tempURI}
-	statement, err := GenerateEndorsement(binaryHash, validity, provenances)
+	reference := verifier.ProvenanceIR{
```

Comment:
Can you add a test with multiple values in `BinarySHA256Digests`?

Reply:
Yes, I'll do that!